uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt311/client/sdkmqtt/Mqtt311ConnectionImpl.java

```diff
@@ -5,6 +5,7 @@
 
 package com.aws.greengrass.testing.mqtt311.client.sdkmqtt;
```

Comment:
The package of the file is invalid.

sdk mqtt means  IoT device SDK (library) is used. 

You use completely another MQTT library.


Again 'AWS IoT Device SDK for Java v2'  is the library which you do not use.
FYI: that is sources of that library: https://github.com/aws/aws-iot-device-sdk-java-v2